src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/EntityDto.java

```diff
@@ -98,7 +98,7 @@ public String getEmail() {
     }
 
     public void setEmail(String email) {
-        this.email = email;
+        this.email = email.trim();
```

Comment:
if we receive null, will cause problems - only trim email if not null?

Reply:
Oh yes. Just seen another unit test break for this too, which is good. I will amend.